### ISSUE TITLE
Events not rendering with vue version 2.5.17

### ISSUE THREAD

**Issue by elliottross23** [antoniandre/vue-cal-v4#64]
**Vue Version:** 2.5.17, generated project with vue cli 3.

I'm trying to do a simple example to get going but events aren't rendering. I'm copying and pasting the custom events render code like this:

```
<vue-cal :events-on-month-view="true" :show-all-day-events="'short'" :events="events">
	<div slot="event-renderer" slot-scope="{ event, view }">
		<v-icon>{{ event.icon }}</v-icon>

		<div class="vuecal__event-title" v-html="event.title" />
		<!-- Or if your events are editable: -->
		<div class="vuecal__event-title" v-html="event.title" />

		<small class="vuecal__event-time">
			<strong>Event start:</strong> <span>{{ event.startTime }}</span>
			<br />
			<strong>Event end:</strong> <span>{{ event.endTime }}</span>
		</small>
	</div>
</vue-cal>
```

My event data
```
events: [
	{
		start: '2019-03-14 06:00',
		end: '2019-03-14 10:00',
		title: 'Golf with John',
	},
],
```

The event just shows up as a gray block....

<img width="1131" alt="Screen Shot 2019-03-13 at 8 19 43 PM" src="https://user-images.githubusercontent.com/7434153/54322926-77adc580-45cd-11e9-8a2d-a64fca3881ba.png">
<img width="330" alt="Screen Shot 2019-03-13 at 8 19 48 PM" src="https://user-images.githubusercontent.com/7434153/54322927-77adc580-45cd-11e9-87b5-6604bd63ec33.png">
<img width="1123" alt="Screen Shot 2019-03-13 at 8 19 58 PM" src="https://user-images.githubusercontent.com/7434153/54322928-77adc580-45cd-11e9-867e-9961152335fb.png">

Any ideas?

**Comment by RoscaS** [comment]
Same here

**Comment by antoniandre** [comment]
Hi @elliottross23,
This will be fixed with a Vue.js update to at least 2.6.7.
I will look further into this issue later today.

**Comment by elliottross23** [comment]
@antoniandre upgrading fixed the issue. Do you want me to leave it open for 2.5.x? I updated the title to be specific.

**Comment by antoniandre** [comment]
Thank you @elliottross23, yes please leave it open.
I have experienced a couple of issues from late vue 2.5.x to 2.6.6 (like #57), I will probably wrap up a working version of vue-cal for vue ≤ 2.6.6 and create a new vue-cal@2.0.0 for vue 2.6.7+

**Comment by antoniandre** [comment]
Hi @elliottross23, @RoscaS, 
I have released version 1.40.4 of vue-cal which is compatible with **Vue.js version 2.5.18 or above**. vue-cal's documentation now states it.

For information, Vue@2.5.17 has an issue here: https://github.com/vuejs/vue/issues/8587.

**Comment by elliottross23** [comment]
@antoniandre Great quick response and great library! Thanks!
